newIDE/app/src/ObjectGroupEditor/index.js

```diff
@@ -114,9 +124,9 @@ const ObjectGroupEditor = ({
             project={project}
             globalObjectsContainer={globalObjectsContainer}
             objectsContainer={objectsContainer}
-            value={newObjectName}
+            value={objectName}
             excludedObjectOrGroupNames={objectsInGroup}
-            onChange={setNewObjectName}
+            onChange={editObject}
```

Comment:
I think this is only to set the textfield value. So you could only use `setObjectName` and not call the `onObjectGroupUpdted`

Reply:
But technically it's a field like others no? Why wouldn't we want to warn if they changed it?

Reply:
oh no you're right, it doesn't make sense